app/server/api/user_api.py

```diff
@@ -20,16 +20,19 @@ class UserAPI(MethodView):
     @requires_auth
     def get(self, user_id):
 
-        can_see_full_details = AccessControl.has_suffient_role(g.user.roles,{'ADMIN': 'admin'})
+        can_see_full_details = AccessControl.has_suffient_role(
```

Comment:
I get the reasoning behind splitting this over multiple lines (PEP8 and all that), but my instinctive response is that unless we're at the 121 line limit, this actually harms readability rather than improves it. open to other opinions though @tristanhcole @RubenGeo @wangfowen

Reply:
I do not have strong opinion about this. I think this was messed up by my linter somewhere